File: 03.Functions-And-Statements-Exercise/factorialDivision.js
function divideFactorials(num1, num2) {

    function calculateFactorial(number) {
        let i = 1
        let result = 1

        while (number != i) {
            result *= i + 1
            i += 1
        }

        return result
    }

    firstNumber = calculateFactorial(num1)
    secondNumber = calculateFactorial(num2)

    console.log((firstNumber / secondNumber).toFixed(2))

}

divideFactorials(5, 2)
divideFactorials(6, 2)

File: 03.Functions-And-Statements-Exercise/factorial_division.py
def divide_factorials(num1, num2):
    def calculate_factorial(number):
        i = 1
        result = 1

        while number != i: 
            result *= i + 1
            i += 1

        return result
    
    first_number = calculate_factorial(num1)
    second_number = calculate_factorial(num2)

    final_result = first_number / second_number

    print(f"{final_result:.2f}")



divide_factorials(5, 2)
divide_factorials(6, 2)